src/actions/integrations.ts

```diff
@@ -52,6 +52,10 @@ export function removeIncomingHook(hookId: string): ActionFunc {
         } catch (error) {
             forceLogoutIfNecessary(error, dispatch, getState);
 
+            dispatch(batchActions([
+                {type: IntegrationTypes.DELETE_INCOMING_HOOK_FAILURE, error},
```

Comment:
Sorry we dont need the failure action here. 

i meant adding back just the log error
`dispatch(logError(error));`

Reply:
thanks, I fixed it